t/plugin/basic-auth.t

```diff
@@ -196,3 +196,34 @@ Authorization: Basic Zm9vOmJhcg==
 hello world
 --- no_error_log
 [error]
+
+
+
+=== TEST 9: invalid schema, only one field `username`
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/consumers',
+                ngx.HTTP_PUT,
+                [[{
+                    "username": "foo",
+                    "plugins": {
+                        "basic-auth": {
+                            "username": "foo"
+                        }
+                    }
+                }]]
+                )
+
+            ngx.status = code
+            ngx.print(body)
```

Comment:
Do we need print?

Reply:
Yes, we need to confirm that the error message is the same as expected.